chef/cookbooks/provisioner/templates/suse/crowbar_register.erb

```diff
@@ -285,8 +285,13 @@ curl -L -o /etc/chef/validation.pem \
     --connect-timeout 60 -s \
     "$HTTP_SERVER/validation.pem"
 
+TMP_ATTRIBUTES=$(mktemp --suffix .json)
+
+# Make sure that we have the right target platform
+echo "{ \"target_platform\": \"$CROWBAR_OS\" }" > "$TMP_ATTRIBUTES"
```

Comment:
does that add it as normal attributes or as default attribute ? in mkcloud we're setting normal.target_platform via knife node edit